src/utils/date-time.js

```diff
@@ -1,19 +1,57 @@
+
+const dayList = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+const monthList = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep',
+  'Oct', 'Nov', 'Dec'];
+const monthLongList = ['January', 'February', 'March', 'April', 'May', 'June',
+  'July', 'August', 'September', 'October', 'November', 'December'];
+
+function DateTimeFormat(locale, options) {
```

Comment:
Do we need to define our own `DateTimeFormat`? Can't we just use the one in `Intl` for `en-US` somehow?